docs/components/rpa/getting-started.md

```diff
@@ -40,11 +43,11 @@ Once you have written your script, you can test it on a local RPA worker.
    2. Review the execution log and the variables created during the script execution within Modeler.
```

Comment:
> **Check Desktop Modeler**: Ensure the RPA worker is now connected to Desktop Modeler. The worker should automatically connect. If not, ensure the connection URL is correct.

How do you ensure the connection URL is correct? Is this under a particular UI component?

Reply:
Yes, this is a UI component. Maybe we can make it clearer how to get to the config view:

> 2. **Check Desktop Modeler**: Ensure the RPA worker is now connected to Desktop Modeler. The worker should automatically connect. If not, click on the connection status to display additional configuration options.

Reply:
This works for me! Went out with my latest commit.